backend/infrahub/core/node/__init__.py

```diff
@@ -731,7 +736,7 @@ async def process_label(self, db: InfrahubDatabase | None = None) -> None:  # no
                 self.label.value = " ".join([word.title() for word in self.name.value.split("_")])
                 self.label.is_default = False
 
-    async def new(self, db: InfrahubDatabase, id: str | None = None, **kwargs: Any) -> Self:
+    async def new(self, db: InfrahubDatabase, id: str | None = None, process_pools: bool = True, **kwargs: Any) -> Self:
```

Comment:
An issue here which has more to do with how the Node class works than the PR is that because of the `**kwargs` stuff we make `process_pools` a reserved keyword that can't be used for attribute or relationship names. There's a `RESERVED_ATTR_REL_NAMES` in the code I think we should add `process_pools` to that list. It will potentially cause issues if anyone is using an attribute with this name. I think it should be fairly safe to do in this instance..

We should refactor the Node class at some point and or get rid of that utterly awful `**kwargs: Any` and instead have a data fields where all the attribute and relationship data goes.

Reply:
Thanks I wasn't aware of this...